### DISCUSSION TITLE
Pull request erstellen

### DISCUSSION THREAD

**Discussion by Linos88** [hoylabs/OpenDTU-OnBattery#869] [General]
Hi,
kann mir jemand einen Tipp geben wie ich ein Pull request erstelle.
Ich bekomme immer folgende Meldung: Pull request creation failed. Validation failed: must be a collaborator

Ich möchte eine allgemeine Anfrage stellen, also nichts spezifisches zu einer Version oder so...

Liegt es an der Einstellung... "master" & "development"
<img width="587" alt="image" src="https://github.com/helgeerbe/OpenDTU-OnBattery/assets/83694034/945ece24-3f37-4ecc-9a0d-ac8c3394929c">

**Comment by schlimmchen** [comment]
Wenn du einen Pull Request erstellen möchtest, um deine Arbeit aus deinem Fork anzubieten, dann musst du deinen Branch aus deinem Fork auf der rechten Seite aufwählen (klicke vorher den Link "compare accross forks"), und der linke Branch muss der development Branch sein.

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/2c65d734-6422-48b6-beb5-5e77e0fa462e)

**Comment by Linos88** [comment]
Ohje, sorry da habe ich was falsch verstanden... 
Ich wollte quasi eine Anfrage stellen, bezüglich hinzufügen
Arbeitsleistung kann ich nur zur verfügung stellen in Form von testen :-)

Wo kann ich am besten eine Anfrage platzieren?

**Reply by schlimmchen** [reply]
Dann erstellt du ein neues "Issue".
